Makefile

```diff
@@ -49,3 +49,12 @@ index:
 index-cleanup:
 	gcloud datastore indexes cleanup index.yaml
 
+build-workers:
+ifndef REGISTRY_PROJECT_IDENTIFIER
+	@echo "Error! REGISTRY_PROJECT_IDENTIFIER must be set."; exit 1
+endif
+	gcloud builds submit --config cmd/capabilities/cloudbuild.yaml \
+    --substitutions _REGISTRY_PROJECT_IDENTIFIER="${REGISTRY_PROJECT_IDENTIFIER}"
+
+deploy-workers:
+	./cmd/capabilities/DEPLOY-WORKERS.sh
```

Comment:
Nit. Maybe it's more clear to group deployment scripts into one place, e.g. under deployments/ folder? 

Also one idea for future improvements: from the user perspective, we could have one Makefile command or a script that can deploy a complete set of Registry applications, including worker + registry backend + database.

Reply:
Done